components/select/index.tsx

```diff
@@ -0,0 +1,23 @@
+import dynamic from "next/dynamic";
+
+const Select = dynamic(() => import("./select"), { ssr: false });
+
+interface Props {
+  allSkills: { name: string; count: number }[];
+  selectedSkill: any;
+  onChange: any;
+}
+
+const SelectComponent: React.FC<Props> = ({
```

Comment:
Proč máme tu komponentu obalenou ještě takhle do jedné vrstvy? Souvisí to nějak s tím `dynamic` nahoře? To je potřeba k čemu?

Reply:
Nezkoumal jsem to do hloubky, ale bylo to potřeba, protože si ta knihovna moc nerozumí s SSR, co jsem tak pochopil. 

Každopádně vyřešil jsem to jiným, lepším způsobem. Celý tohle obalení jsem zahodil a stačilo jen tomu selectu nastavit atribute `instanceId`. :)